Week3/BusinessCard/app/src/main/java/com/janinculhaoglu/businesscard/MainActivity.kt

```diff
@@ -0,0 +1,68 @@
+package com.janinculhaoglu.businesscard
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.animation.AnimationUtils
+import android.widget.Button
+import android.widget.TextView
+
+class MainActivity : AppCompatActivity() {
+
+    private val experienceList = listOf(
+        "Creation of a native iOS app to find recipes by ingredient",
+        "Creation of an app to help you find eco-friendly places nearby",
+        "Creation of an android business card app"
+    )
```

Comment:
I suggest that you take this dummy data outside of the Activity. For example you can create a file that just holds this dummy data and then in the activity you would just ask for it.

Activity should be responsible only for displaying data. I'm not taking this as an error since this your first Android app, just giving out suggestions.